casepro/contacts/models.py

```diff
@@ -430,7 +440,7 @@ def as_json(self, full=True):
         """
         Prepares a contact for JSON serialization
         """
-        result = {'id': self.pk, 'name': self.get_display_name()}
+        result = {'id': self.pk, 'name': self.get_display_name(), 'urns': self.get_display_urns()}
```

Comment:
The compact view is the representation that's currently available on the case page which is one of the places we want the urns to be visible. I'll change the case_read view to retrieve the urns instead.